packages/preview-middleware-client/src/adp/controllers/BaseDialog.controller.ts

```diff
@@ -57,7 +61,8 @@ export default abstract class BaseDialog extends Controller {
 
         const updateDialogState = (valueState: ValueState, valueStateText = '') => {
             input.setValueState(valueState).setValueStateText(valueStateText);
-            beginBtn.setEnabled(valueState === ValueState.Success);
+            // fix for ui5 version 1.71 (button does not invalidate and rerender)
+            beginBtn.setEnabled(valueState === ValueState.Success).rerender();
```

Comment:
Is it possible to call this only if UI5 version is 1.71?

Reply:
Removed this: [fe9fd7f](https://github.com/SAP/open-ux-tools/pull/1597/commits/fe9fd7fb841bc9c29b7f91b0e9507660b78b15fb)